Tests/LibWeb/Text/input/css/support/url-substitution-base.css

```
@@ -0,0 +1,35 @@
+    --nested-url: url("img.png");
+
+    & #nested {
+        background-image: var(--nested-url);
+    }
+}
+
+#bare {
+    & #bare-nested {
+        color: blue;
+    }
+    background-image: var(--url);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)